src/extension/bitcoin.rs

```diff
@@ -214,6 +214,10 @@ impl extension::Jet for BtcNode {
         }
     }
 
+    fn wmr(&self) -> Cmr {
+        self.cmr()
+    }
```

Comment:
In ab95a4a54d4a800d9a3979732f74375c83747ba4:

...I guess, then this would need an explicit conversion, but that would be clearer IMO than this, which I thought was a mistake or placeholder at first.